modules/bibformat/lib/bibreformat.py

```diff
@@ -280,7 +279,7 @@ def outdated_caches(fmt, last_updated, chunk_size=5000):
     max_id = run_sql("SELECT max(id) FROM bibrec")[0][0] or 0
     for start in xrange(1, max_id + 1, chunk_size):
         end = start + chunk_size
-        recids += intbitset(run_sql(sql, (last_updated_str, fmt, start, end)))
```

Comment:
Since the `last_updated` is not used anymore, maybe it could be a nice idea to remove it from the parameters.